scripts/get_hashes_page.py

```diff
@@ -18,6 +18,7 @@ def generate_doc_file(cmp_version, contracts):
 https://crates.io/crates/cairo-lang-compiler/{cmp_version}[cairo {cmp_version}]
 """
     hashes = "// Class Hashes\n"
+    contracts['contracts'].sort(key=lambda x: x['name'])
```

Comment:
isn't this doing lexicographical sort?

Reply:
Yes

Reply:
but weren't we favoring category grouping over alphabetical order?

Reply:
We did open an issue for that, #934, but we are currently sorting alphabetically.

Reply:
i guess find it a bit weird to open a PR with no associated issue that goes in a different direction than the issue we do have. instead i guess i would've addressed the issue directly or just leave it as it is, since i'm not sure alphabetical order is better than [just whatever order](https://github.com/OpenZeppelin/cairo-contracts/pull/937#discussion_r1514871729) until we address it. i guess it's not harming so i wouldn't revert this either

Reply:
i see now that the PR is also fixing the `v` in the CI. we should mention that in the issue/commit